python/src/triton.cc

```diff
@@ -1997,7 +1997,8 @@ void init_triton_translation(py::module &m) {
   m.def(
       "get_warp_size",
       []() {
-        return std::get<1>(getArchInfo());
+        if (userDefArch.empty())
```

Comment:
How did you get this compiled? I can't find its definition or initialization.

Also I still recommend making another PR to enable `MI_GPU_ARCH`, because it needs to address quite a few considerations, for example, the name of the variable itself is questionable due to potential naming conflicts.

Reply:
Accidental change of user defined arch. sorry,

I'll create PR to enable user defined arch today or tomorrow